Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2033,82 +2098,20 @@ def parse_incident_from_item(item):
-                            attached_email_headers.append((h, v))
-                        for header in attachment.item.headers:
-                            if (
-                                    (header.name, header.value)
-                                    not in attached_email_headers
-                                    and header.name != "Content-Type"
-                            ):
-                                attached_email.add_header(header.name, header.value)
-                    attached_email_bytes = attached_email.as_bytes()
-                    chardet_detection = chardet.detect(attached_email_bytes)
-                    encoding = chardet_detection.get('encoding', 'utf-8') or 'utf-8'
-                    try:
-                        # Trying to decode using the detected encoding
-                        data = attached_email_bytes.decode(encoding)
-                    except UnicodeDecodeError:
-                        # In case the detected encoding fails apply the default encoding
-                        demisto.info(f'Could not decode attached email using detected encoding:{encoding}, retrying '
-                                     f'using utf-8.\nAttached email:\n{attached_email}')
-                        data = attached_email_bytes.decode('utf-8')
-
-                    file_result = fileResult(
-                        get_attachment_name(attachment.name) + ".eml",
-                        data,
-                    )
-
-                if file_result:
-                    # check for error
-                    if file_result["Type"] == entryTypes["error"]:
-                        demisto.error(file_result["Contents"])
-                        raise Exception(file_result["Contents"])
-
-                    # save attachment to incident
-                    incident["attachment"].append(
-                        {
-                            "path": file_result["FileID"],
-                            "name": get_attachment_name(attachment.name) + ".eml",
-                        }
-                    )
-
-            labels.append(
-                {
-                    "type": label_attachment_type,
-                    "value": get_attachment_name(attachment.name),
-                }
-            )
-            labels.append(
-                {"type": label_attachment_id_type, "value": attachment.attachment_id.id}
-            )
+                labels.append(
+                    {
+                        "type": label_attachment_type,
+                        "value": get_attachment_name(attachment.name),
+                    }
+                )
+                labels.append(
+                    {"type": label_attachment_id_type, "value": attachment.attachment_id.id}
+                )
+            except ErrorCannotOpenFileAttachment:
+                demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
```

Comment:
```suggestion
                demisto.error("Could not open attachment file for message_id: {}".format(item.message_id))
```

Reply:
See my comment in the issue.
If we proceed in this direction with exception, we should catch the exception on single attachment level.
If there are 5 attachments in the email, only one of them should be dropped